Fitter/TauES/createinputsTES.py

```diff
@@ -88,10 +116,17 @@ def main(args):
     #   BINS / FIT REGIONS  #
     ############
       
-    bins = [ Sel("baseline", setup["baselineCuts"]) ]
+    bins = [ ]
+    if DM == '':
+      bins.append(Sel("baseline", setup["baselineCuts"]))
+    jetcut = map_wp_to_int["againstjet"][againstjet]
+    electroncut = map_wp_to_int["againstelectron"][againstelectron]
+    setup["baselineCuts"] = setup["baselineCuts"].replace('idDeepTau2018v2p5VSjet_2>=5', f'idDeepTau2018v2p5VSjet_2>={jetcut}')
```

Comment:
In this way you are "overwriting" the selection made in the configuration file, right ? 
Although passing an argument for the working points seems indeed more convenient, maybe it defeats the purpose of the configuration file? Not sure about it, @IzaakWN what do you think? 
In any case, you should explain how to run the scripts with the new arguments in the README :)

Reply:
Argument is passed [here](https://github.com/saswatinandan/TauFW/blob/master/Fitter/TauES/createinputsTES.py#L224-L225). It will replace the default vale with the argument value. Since these parameter values are set to default one, one doesn't need to change anything but if someone needs to change this default value, he/she can pass the desired value like Tight, Medium.